### PR TITLE
Enhance installation instructions for MacOS

### PR DESCRIPTION
**Pull request by katafrakt**
More information about building Amber from source on MacOS.

* Correct heading level
* Add info about solving issues while building from source

### CODE REVIEW

File: guides/installation.md [hunk 1]
```diff
@@ -71,3 +71,16 @@ Installing Amber with these package managers also installs Crystal.
   sudo port selfupdate
   sudo port install amber
   ```
+
+### From source
+
+Known problems while trying to build Amber from source on MacOS:
+
+- If you see linker problems, such as `ld: library not found for -lssl (this usually means you need to install the development package for libssl)`, you
```
Comment (drujensen) [inline guides/installation.md]: :+1:

File: guides/installation.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ sudo swapon /swapfile
 
 You should now be able to run `amber` in the command line.
 
-#### macOS
+## MacOS
```
Comment (drujensen) [inline guides/installation.md]: I think it's official name is lowercase `macOS` but probably better capitalized here.

Comment (katafrakt) [reply guides/installation.md]: ooops, I can change it to lowercase if you think it's better

Comment (drujensen) [reply guides/installation.md]: Yes please.  I'm just bikeshedding.  lol.

**Pull request merged**